docs/operating-eck/installing-eck.asciidoc

```diff
@@ -0,0 +1,88 @@
+:page_id: installing-eck
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+[id="{p}-{page_id}"]
+= Install ECK 
+
+{n} (ECK) is a link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes operator] to orchestrate Elastic applications ({eck_resources_list}) on Kubernetes. It relies on a set of link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions[Custom Resource Definitions (CRD)] to declaratively define the way each application is deployed. CRDs are global resources shared by all users of the Kubernetes cluster, which requires an administrator to install them for initial use. The operator itself can be installed as a cluster-scoped application or it can be restricted to a handful of namespaces. Multiple copies of the operator can be installed on a single Kubernetes cluster provided that the global CRDs are compatible with each instance and optional cluster-scoped extensions such as the <<{p}-webhook,validating webhook>> are disabled.
```

Comment:
"handful" confused me, it sounds like you can restrict only to very few namespaces.